exp-models/addon/models/account.js

```diff
@@ -2,14 +2,34 @@
 Manage data about one or more documents in the accounts collection
  */
 
+import moment from 'moment';
```

Comment:
Feels creepy not to add the import statement here, and the ember-moment docs show this as an example: https://github.com/stefanpenner/ember-moment#es6-moment

So probably no.